api/src/server.ts

```diff
@@ -31,8 +31,9 @@ app.get("/api/terms/:termId/courses", notangles.getCourseList);
 app.get("/api/terms/:termId/freerooms", freerooms.getFreeroomsData);
 
 app.get("/api/availableterm/", getAvailableTermName);
+app.get("/api/availablestartdate/", notangles.getLatestStartDate);
 app.get("/api/currentterm/", getCurrentTermName);
-app.get("/api/startdate/:client", getStartDateByRouteParams);
+app.get("/api/currentstartdate/", getCurrentTermDate);
```

Comment:
this requires a client param (notangles) in our case will this work ?
is ``getStartDateByRouteParams`` completely removed?
If it is not necessary, we don't need to keep it anymore since notangles using it only. (Or are u just ceebs atm)

Reply:
yep we can remove that file, just kept it incase we needed in the future; but I think that would be unlikely